src/test/java/org/opentripplanner/routing/edgetype/loader/HopFactoryTest.java

```diff
@@ -39,8 +38,6 @@ public class HopFactoryTest {
 
```

Comment:
According to #4105 this test can be removed, so lets put it out of its misery.

RIP.